docs/masking-providers.md

```diff
@@ -733,6 +734,36 @@ date at */dateWritten*.
 | preserveDomains | Integer  | Number of domains to preserve starting from the right; if value = -1 then all domains will be preserved (that is, everything after \@) | 1                 |
 | nameLength      | Integer  | Length of username to generate; if value = -1, then the username will be 5 to 8 characters long. 0 is invalid      | \-1               |
 
+#### FHIR_MORTALITY_DEPENDENCY
+
+This provider enforces a specific type of dependency between the birth date and the deceased datetime and deceased boolean properties that exist in a FHIR Patient resource, which is a very specific schema imposed on a JSON document. 
```

Comment:
"This provider enforces a specific type of dependency between the birth date and the deceased datetime and deceased boolean properties"

Is this provider enforcing a specification in FHIR?  If so, can you provide a link to the part of the specification this is trying to enforce?

Reply:
This is not a specifc FHIR specification.  It is based on looking at the current FHIR 4 documentation about what the fields are and what their content is.  The interaction between the fields is based on the customer request.